docs/en/setup/backend/exporter.md

```diff
@@ -81,6 +81,8 @@ exporter:
     # Kafka producer config, JSON format as Properties.
     kafkaProducerConfig: ${SW_EXPORTER_KAFKA_PRODUCER_CONFIG:""}
     kafkaTopicTrace: ${SW_EXPORTER_KAFKA_TOPIC_TRACE:skywalking-trace}
+    # Trace filter
+    kafkaTraceFilterError: ${SW_EXPORTER_KAFKA_TRACE_FILTER_ERROR:false}
```

Comment:
```suggestion
    exportErrorStatusTraceOnly: ${SW_EXPORTER_KAFKA_TRACE_FILTER_ERROR:false}
```

Notice, this is a filter at codes, but from user perspective, I would say this is an exporter configuration.

Please update the codes and docs according to the new propose name.

Reply:
get it

Reply:
> And you should update this doc to describe this feature, rather than only a config item

any template for it ?

Reply:
I updated for you.